pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/pubsub/DSMtasksSubscription.java

```diff
@@ -62,7 +62,7 @@ public static Subscriber subscribeDSMtasks(String projectId, String subscription
                             doESExport(attributesMap, data);
                             break;
                         case PARTICIPANT_REGISTERED:
-                            generateStudyDefaultValues(consumer, attributesMap);
+                            generateStudyDefaultValues(consumer, attributesMap, data);
```

Comment:
Didn't realize that DSM was never looking into payload for "PARTICIPANT_REGISTERED" events and only looking into payload for UPDATE_CUSTOM_WORKFLOW published by DSS. Looks like DSS was always publishing empty payload for PARTICIPANT_REGISTERED.